nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-nar-utils/src/main/java/org/apache/nifi/nar/NarClassLoader.java

```diff
@@ -205,26 +216,41 @@ private void updateClasspath(File root) throws IOException {
     }
 
     @Override
-    protected String findLibrary(final String libname) {
+    public String findLibrary(String libname) {
```

Comment:
Is there a reason here to override the method just to call the superclass definition, instead of just removing the method from NarClassLoader?

Reply:
Gotcha. Thanks for clarifying. I know there are sometimes some oddities like this in Java that do require overriding a method and calling the superclass. But 80% of the time when I do it, it's because I refactored and it just didn't even occur to me to delete it. Just figured I'd check.